libecl_well/tests/well_state_load_missing_RSEG.c

```diff
@@ -50,13 +51,13 @@ int main(int argc , char ** argv) {
     for (global_well_nr = 0; global_well_nr < header->nwells; global_well_nr++) {
       well_state_type * well_state = well_state_alloc(well_name , global_well_nr , open , type , report_nr , valid_from);
       test_assert_true( well_state_is_instance( well_state) );
-      well_state_add_connections( well_state , grid , rst_file , 0 );
-      well_state_add_MSW( well_state , rst_file , global_well_nr , load_segment_information );
+      well_state_add_connections2( well_state , grid , rst_view , 0 );
```

Comment:
Yes - but I think Resinsight is using the old `well_state_add_connections()` - so a rename was warranted :-(